packages/code-du-travail-api/tests/fiches_ministere_travail_data.json

```diff
@@ -0,0 +1,20 @@
+[{
+  "date": "23/05/2017",
```

Comment:
once rebased, this document should be generated using the script that update mock data